src/config/saml.php

```diff
@@ -16,7 +16,7 @@
     /*
      * The path to the working phpsimplesaml install
      */
-    'sp_path' => "/var/simplesamlphp",
+    'sp_name' => env("SP_NAME", "default_sp"),
```

Comment:
This is a different variable. SP name is the name of something inside simplesamlphp, the path is the path to it.